wordvecspace/command.py

```diff
@@ -1,12 +1,20 @@
 import code
 
 from basescript import BaseScript
-from convert import GWVec2WordVecSpace
-from wordvecspace import WordVecSpace
+from .convert import GWVec2WordVecSpace
+from .wordvecspace import WordVecSpace
+from .tornado_wvs import *
```

Comment:
@RamanjaneyuluIdavalapati Bad idea to use "import *". We must try our best to avoid this and it becomes difficult to debug where a symbol came from. Also, please call `tornado_wvs` => `service.py`. The fact that we are using `tornado` is not important in this file.

Reply:
@prashanthellina done